pyccl/core.py

```diff
@@ -21,13 +22,15 @@
     'boltzmann_class': lib.boltzmann_class,
     'boltzmann_camb': lib.boltzmann_camb,
     'boltzmann_isitgr': lib.boltzmann_isitgr,
+    'calculator': lib.pklin_from_input
 }
 
 matter_power_spectrum_types = {
     'halo_model': lib.halo_model,
     'halofit': lib.halofit,
     'linear': lib.linear,
-    'emu': lib.emu
+    'emu': lib.emu,
+    'calculator': lib.pklin_from_input
```

Comment:
This line is wrong. It should be `pknl_from_input`.

Reply:
Fixed